packages/lesswrong/components/vulcan-forms/FormGroup.tsx

```diff
@@ -47,9 +48,10 @@ const FormGroupHeaderComponent = registerComponent('FormGroupHeader', FormGroupH
 export const groupLayoutStyles = (theme: ThemeType): JssStyles => ({
   formSection: {
     fontFamily: theme.typography.fontFamily,
-    border: theme.palette.border.grey400,
+    border: theme.palette.border.grey300,
```

Comment:
@agnesstenlund I believe this border color change matches your designs, but to me the borders feel too faint now (at least on my screen), like it's hard to distinguish them. No worries if you want to keep it this color, but just thought I'd give my 2 cents.

<img width="691" alt="Screen Shot 2023-02-15 at 6 12 00 PM" src="https://user-images.githubusercontent.com/9057804/219212392-c63b8b2a-e8e5-42ad-9363-587932618c6a.png">